source/extensions/filters/http/jwt_authn/filter_factory.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "envoy/config/filter/http/jwt_authn/v2alpha/config.pb.h"
+#include "envoy/server/filter_config.h"
+
+#include "extensions/filters/http/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+class FilterFactory : public Server::Configuration::NamedHttpFilterConfigFactory {
```

Comment:
If you inherit from the newly added `FactoryBase`, you can reduce most of the boiler plate here. This is hot off-the-press :)

Reply:
Done